src/ExportJobs/apiQuery.test.js

```diff
@@ -36,7 +36,12 @@ describe('Export jobs api queries', () => {
         }),
       });
 
-      const { result, waitFor } = renderHook(() => useExportJobsQuery(), { wrapper });
+      const { result, waitFor } = renderHook(() => useExportJobsQuery(
```

Comment:
Is it duplicate? I see the same on line 70. Could you please check?

Reply:
I checked it, and it's not duplicate.